pkg/awsclient/mock/client_generated.go

```diff
@@ -1,7 +1,6 @@
 // Code generated by MockGen. DO NOT EDIT.
 // Source: ./client.go
 
-// Package mock is a generated GoMock package.
```

Comment:
None of the changes to the mock files should be needed.

Reply:
I had installed latest version of mockgen, I think it converted the files, will remove them

Reply:
Removed